Autopilot/boardfiles/Autopilot_ZeropilotRev1.ioc

```diff
@@ -71,55 +72,59 @@ Mcu.Pin15=PA0/WKUP
 Mcu.Pin16=PA1
 Mcu.Pin17=PA2
 Mcu.Pin18=PA3
-Mcu.Pin19=PB0
+Mcu.Pin19=PA4
 Mcu.Pin2=PE5
-Mcu.Pin20=PB1
-Mcu.Pin21=PF14
-Mcu.Pin22=PF15
-Mcu.Pin23=PG0
-Mcu.Pin24=PG1
-Mcu.Pin25=PE7
-Mcu.Pin26=PE13
-Mcu.Pin27=PB10
-Mcu.Pin28=PB11
-Mcu.Pin29=PB12
+Mcu.Pin20=PA5
```

Comment:
really unsure of what's going on here.... Kinda making me nervous.

Reply:
Me too actually, Ill investigate this further